cluster/src/main/java/org/apache/iotdb/cluster/log/applier/AsyncDataLogApplier.java

```diff
@@ -131,7 +132,7 @@ private PartialPath getPlanKey(PhysicalPlan plan) throws StorageGroupNotSetExcep
 
   private PartialPath getPlanSG(PhysicalPlan plan) throws StorageGroupNotSetException {
     PartialPath sgPath = null;
-    if (plan instanceof InsertPlan) {
+    if (plan instanceof InsertPlan && !(plan instanceof InsertMultiTabletPlan)) {
```

Comment:
This forces InsertMultiTabletPlan to be executed serially with other plans, converts the applying process to a synchronized one, and within one InsertMultiTabletPlan, tablets of different storage groups are executed one by one, so the parallelism is significantly reduced.
I think we can add a limitation that each InsertMultiTabletPlan after splitting will only contain tablets of one storage group, so tablets of different storage groups can be inserted in parallel.

Reply:
Sure, I'll try to do this